profiles/common/modules/custom/nexteuropa_token/modules/nexteuropa_token_ckeditor/nexteuropa_token_ckeditor.module

```diff
@@ -45,11 +52,20 @@ function nexteuropa_token_ckeditor_menu() {
   $items = array();
   $items['nexteuropa/token-ckeditor/%'] = array(
     'page callback' => 'nexteuropa_token_ckeditor_ajax_callback',
-    'page arguments' => array(2, 3),
+    'page arguments' => array(2),
     'access callback' => 'nexteuropa_token_ckeditor_ajax_access_callback',
     'type' => MENU_CALLBACK,
     'delivery callback' => 'ajax_deliver',
   );
+
+  $items['nexteuropa/token-bean-ckeditor/%'] = array(
+    'page callback' => 'nexteuropa_token_ckeditor_bean_ajax_callback',
+    'page arguments' => array(2),
+    'access callback' => 'nexteuropa_token_ckeditor_bean_ajax_access_callback',
+    'type' => MENU_CALLBACK,
+    'delivery callback' => 'ajax_deliver',
+  );
+
```

Comment:
AJAX ? You mean, for the list using views ?

They want to be able to add any beans in content... to me it doesn't looks so crazy and there are no other way to do it... unless you have a better idea ?